wgpu/gui/qt.py

```diff
@@ -13,50 +14,109 @@
     if libname in sys.modules:
         QtCore = importlib.import_module(libname + ".QtCore")
         widgets_modname = "QtGui" if QtCore.qVersion()[0] == "4" else "QtWidgets"
-        QWidget = importlib.import_module(libname + "." + widgets_modname).QWidget
+        QtWidgets = importlib.import_module(libname + "." + widgets_modname)
         break
 else:
     raise ImportError(
         "Import one of PySide2, PyQt5 before the WgpuCanvas to select a Qt toolkit"
     )
 
+# Enable high-res displays
+try:
+    ctypes.windll.shcore.SetProcessDpiAwareness(2)
+except Exception:
+    pass  # fail on non-windows
+try:
+    QtWidgets.QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling, True)
+    QtCore.QCoreApplication.setAttribute(QtCore.Qt.AA_UseHighDpiPixmaps, True)
+except Exception:
+    pass  # fail on older Qt's
```

Comment:
I feel like this code should be in a method.. Qt app authors probably want control over these attributes themselves.

Reply:
Removed `AA_UseHighDpiPixmaps` as it is notr strictly needed for wgpu.